File: src/main/java/ru/romangr/catbot/handler/HelpCommandHandler.java
package ru.romangr.catbot.handler;

import lombok.RequiredArgsConstructor;
import ru.romangr.catbot.executor.action.TelegramAction;
import ru.romangr.catbot.executor.action.TelegramActionFactory;
import ru.romangr.catbot.telegram.model.Chat;

import java.util.List;

@StaticCommand(BotCommand.HELP)
@RequiredArgsConstructor
public class HelpCommandHandler extends StaticCommandHandler {

    private static final String HELP_STRING = "Type /cat to get a random cat :3";

    private final TelegramActionFactory actionFactory;

    @Override
    protected List<TelegramAction> handleCommand(Chat chat, String messageText) {
        return List.of(actionFactory.newSendMessageAction(chat, HELP_STRING));
    }
}

File: src/main/java/ru/romangr/catbot/handler/HelpCommandHandler.kt
package ru.romangr.catbot.handler

import ru.romangr.catbot.executor.action.TelegramAction
import ru.romangr.catbot.executor.action.TelegramActionFactory
import ru.romangr.catbot.statistic.StatisticService
import ru.romangr.catbot.telegram.model.Chat

@StaticCommand(BotCommand.HELP)
class HelpCommandHandler(private val actionFactory: TelegramActionFactory,
                         timeToSendMessageToSubscribers: Int,
                         statisticService: StatisticService)
    : StaticCommandHandler(statisticService) {

    private val helpMessage =
            """/cat to get a random cat 🐱
/subscribe to get a random cat every day 🐈 ($timeToSendMessageToSubscribers:00 UTC)"""

    override fun handleStringCommand(chat: Chat, text: String): List<TelegramAction> {
        return listOf(actionFactory.newSendMessageAction(chat, helpMessage))
    }

}
